lips/lip-6.mdx

```diff
@@ -10,27 +10,26 @@ created: 07/30/2020
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
 # Summary
-This LIP describes the conceptual governance model of the Libra blockchain, in particular, the on-chain Validator Set configuration.
-Validator Set is part of a global on-chain configuration stored as part of the blockchain state, it additionally includes the list of registered currencies, publishing policies and Libra version.
-Validator Set defines a collection of entities, called Validator Nodes, that coordinate and reach consensus on the state of the blockchain by processing transactions, they ensure correct updates to the state even if some Validator Nodes (fewer than a third) fail in arbitrary ways. The LibraRoot account governed by the Libra Payment Network and the Libra Association defines the composition of the validator set. LibraRoot may add or remove members over time.
+This LIP describes the organization of Libra's proof of authority consensus model by which the Libra blockchain agrees upon ledger state. In a proof of authority consensus model, well-known participants leverage cryptographical digital signatures to agree upon a set of transactions and their output to advance the blockchain's state. The set of potential entities participating in consensus are known as Validators, while the active participants are known as the Validator Set. The Validator Set is part of a global on-chain configuration stored as part of the ledger state. Other examples of on-chain configuration include the list of registered currencies, publishing policies and Libra version. LibraRoot is the sole entity capable of adding or removing Validators and specifying the current Validator Set.
```

Comment:
The summary seems to touch on 3 things: proof of authority, Validators/Validator Set and on-chain configuration. I'd perhaps then:
- Move the last sentence (about LibraRoot being the entity to add/remove Validators) and put that before the sentence about the on-chain configuration (otherwise talking about on-chain configurations seems to break the flow). 
- I'd also clarify that every time the validator set changes, a new epoch is marked.
- We should probably describe (in a few words) what LibraRoot is.

For example:

... The set of potential entities participating in consensus are known as Validators, while the active participants are known as the Validator Set. Every time the Validator Set changes, e.g., a new Validator is added to the set, a new blockchain epoch is marked. The only entity capable of adding or removing Validators to the Validator Set is the LibraRoot: a special account on the Libra blockchain capable of performing privileged operations. To represent the current Validator Set in Libra, it is part of a global on-chain configuration ...

Reply:
why bring up epoch change this early, it is somewhat of an implementation detail, no?

and yes, this is super dense.